private/tools/java/com/github/bazelbuild/rules_jvm_external/maven/MavenPublisher.java

```diff
@@ -106,6 +111,17 @@ public static void main(String[] args)
         futures.add(upload(repo, credentials, coords, "-javadoc.jar", docJar, gpgSign));
       }
 
+      if(args.length > 9) {
+        String ext = com.google.common.io.Files.getFileExtension(args[9]);
+        List<String> extraArtifactTuples = Splitter.onPattern(",").splitToList(args[9]);
+        for(String artifactTuple: extraArtifactTuples) {
+          String[] splits = artifactTuple.split("=");
+          String classifier = splits[0];
+          Path artifact = getPathIfSet(splits[1]);
```

Comment:
The `Path` should exist. If it doesn't, we've got real problems.... `Paths.get` is fine here.